pkgs/tools/security/sudo/default.nix

```
@@ -58,7 +65,7 @@ stdenv.mkDerivation rec {
 
   postInstall =
     ''
-    rm -f $out/share/doc/sudo/ChangeLog
+      rm -f $out/share/doc/sudo/ChangeLog
```

```suggestion
      rm $out/share/doc/sudo/ChangeLog
```
we want to notice when the file is missing.